tomls/omnibus-arbitrum-sepolia/perps/markets/ldo.toml

```
@@ -0,0 +1,119 @@
+[var.perps_ldo_market_settings]
+ldo_perps_market_id = "5900"
+ldo_perps_market_skew_scale = "<%= parseEther(String(21_000_000)) %>"
+ldo_perps_market_max_funding_velocity = "<%= parseEther('9') %>"
+ldo_perps_maker_fee_ratio = "<%= parseEther('0.000000') %>"
+ldo_perps_taker_fee_ratio = "<%= parseEther('0.001000') %>"
+ldo_perps_max_market_size = "<%= parseEther(String(695_300)) %>"
+ldo_perps_max_market_value = "<%= parseEther(String(500_000)) %>"
+ldo_perps_initial_margin_ratio = "<%= parseEther('1.25') %>"
+ldo_perps_maintenance_margin_scalar = "<%= parseEther('0.34') %>"
+ldo_perps_minimum_initial_margin_ratio = "<%= parseEther('0.05') %>"
+ldo_perps_flag_reward_ratio_d18 = "<%= parseEther('0.0003') %>"
+ldo_perps_max_liquidation_limit_accumulation_multiplier = "<%= parseEther('1.5') %>"
+ldo_perps_max_seconds_in_liquidation_window = "30"
+ldo_perps_minimum_position_margin = "<%= parseEther('50') %>"
+ldo_perps_locked_oi_ratio = "<%= parseEther('0.5') %>"
+ldo_perps_max_liquidation_pd = "<%= parseEther('0.0005') %>"
+ldo_perps_endorsed_liquidator = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
+
+[invoke.createLdoPerpsMarket]
+target = ["perpsFactory.PerpsMarketProxy"]
+fromCall.func = "owner"
+func = "createMarket"
+args = ["<%= settings.ldo_perps_market_id %>", "Lido dao", "LDO"]
```

```suggestion
args = ["<%= settings.ldo_perps_market_id %>", "Lido DAO", "LDO"]
```

Actual name is `Lido DAO Token` though